src/components/JSONContainer/JSONContainer.tsx

```diff
@@ -3,17 +3,24 @@ import { TableContainer, Button } from "@mui/material";
 import ContentCopyIcon from "@mui/icons-material/ContentCopy";
 import JSONPretty from "react-json-pretty";
 
-function JSONContainer(data: any) {
+interface IJSONProps {
+  data: any;
+}
+
+function JSONContainer(props: IJSONProps) {
```

Comment:
Just a note that the reason I keep the raw (prettified) JSON response under most of the tables for v2 is because it's early days and we might get data updates with more useful info some time in the future, that we're not aware of. We can probably remove it if we're satisfied with the data structure later on.

Reply:
I think this is a good feature that you could see the raw JSON in a pretty mode? If this is getting too long, we could hide this instead of removing it from the page.